### DISCUSSION TITLE
Homebridge support for Eufy Smart lock T8520 ?

### DISCUSSION THREAD

**Discussion by gowtham1643** [samemory/homebridge-eufy-security-control#323] [Q&A]
Any idea when there will be a support for Eufy Smart lock (T8520) ?

**Comment by cturner** [comment]
I suspect this is predicated in support in the eufy-security-client library first.  I also have this lock and am working on a PR to add support to the aforementioned project.  If I can get that working I can probably add it to the homebridge project as well.  Following in case someone beats me to it.

**Reply by gowtham1643** [reply]
Thanks Chris

**Reply by fullphat** [reply]
> ... am working on a PR to add support to the aforementioned project...

Could you walk me through creating a PR?  I'm trying to do one for this plugin but I've never done one before.

**Comment by jredondox** [comment]
would love to get support for this lock as well :)

**Reply by lenoxys** [reply]
depending on @bropat library as soon it's release we pre-dev a lock accessory and should allow us to quickly provide support on lock device.

**Reply by MouseRatt** [reply]
Bropat just updated library for this!

**Reply by jredondox** [reply]
@MouseRatt 
> Bropat just updated library for this!

where is the update? i dont see it

**Reply by MouseRatt** [reply]
It's the library the plug-in is built on.. will have to wait for Homebridge plug-in update

**Reply by jredondox** [reply]
> It's the library the plug-in is built on.. will have to wait for Homebridge plug-in update

Ah got it… hoping tit drops soon 👍
